src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -5750,8 +5750,8 @@ class @var
             // extra check on var
             var x123Var = (DeclarationExpressionSyntax)x1.Parent.Parent;
             Assert.Equal("var", x123Var.Type.ToString());
-            Assert.Null(model.GetTypeInfo(x123Var.Type).Type);
-            Assert.Null(model.GetSymbolInfo(x123Var.Type).Symbol); // The var in `var (x1, x2)` has no symbol
+            Assert.Equal("(Script.var x1, (Script.var x2, Script.var x3))", model.GetTypeInfo(x123Var.Type).Type.ToTestDisplayString());
```

Comment:
>(Script.var x1, (Script.var x2, Script.var x3)) [](http://example.com/codeflow?start=26&length=47)

This doesn't feel right. It looks like `var` binds to ```class @var``` rather than getting inferred.

Reply:
@AlekseyTs Given the existing diagnostic on the test `ERR_DeconstructionVarFormDisallowsSpecificType`, I think `var` is already bound to `class @var` and is not inferred.

Reply:
>  I think `var` is already bound to `class @var` and is not inferred.

Exactly my point. Why do we show it as a tuple type then?

Reply:
Oh it should have been just `Script.var`?
I'll take a look and see why is this the current behavior.

Reply:
@AlekseyTs For this case, the expectation for getting `var` info is to be `Script.var`. But what about the declaration expression as a whole (var + parenthesized designation)? Should it still be `Script.var`? or should it be the tuple?

Reply:
For the whole declaration expression, the type info shouldn't change in this PR. Whatever compiler thinks its type, that should be returned. I am guessing it is a tuple type today.